recipes/parsplice/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "parsplice" %}
+{% set version = "1.1.1" %}
+{% set build = 0 %}
+{% set build_string = "py{}_{}_{}".format(environ.get('CONDA_PY', ''), mpi, build) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/exaalt/parsplice/-/archive/master/parsplice-master.tar.gz
+  sha256: b811db17a0a56685f454e00b665454dd216a722e7ef127c85681ecc1cd23d5e1
+
+build:
+  number: {{ build }}
+  string: {{ build_string }}
+  skip: true  # [not linux]
```

Comment:
Have you tried building for macOS? If it’s hard it doesn’t matter but it’s nice to have if it just works.

Reply:
There is some issue with linking to the lammps library on mac os X. I might try it later on, but for now I would prefer to only release the linux version.